duckduckgo/iOS
# Clean up scripts

Task/Issue URL: https://app.asana.com/0/1203301625297703/1203932773471353/f
CC:

**Description**:
Clean up the root folder and move scripts into a dedicates "scripts" directory.
Get rid of some shell check warnings.
Remove unused scripts.

**Steps to test this PR**:
1. Run every script to make sure it works.
2. Run the app to make sure it works.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
